dist/scss/Components/ExternalNavBar/NavBar/style.scss

```diff
@@ -0,0 +1,12 @@
+@import './../../../vars';
+
+.external-navbar-of-trust {
+    display: flex;
+    width: 50%;
+    justify-content: space-evenly;
+    padding: $MARGIN 0;
```

Comment:
The idea with all caps variables was that they are kind of basic blocks that you will use to build other variables and not use directly.

e.g.
In vars.scss

```
$MARGIN: 1rem;
$PADDING: 1rem;
$default-padding: $PADDING * 2 0;
$default-margin: $MARGIN * 3 0;
```

Reply:
Why this is this marked as resolved?